glib/src/auto/markup_parse_context.rs

```diff
@@ -0,0 +1,101 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::translate::*;
+use crate::Error;
+use std::mem;
+use std::ptr;
+
+crate::wrapper! {
+    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct MarkupParseContext(Shared<ffi::GMarkupParseContext>);
+
+    match fn {
+        ref => |ptr| ffi::g_markup_parse_context_ref(ptr),
+        unref => |ptr| ffi::g_markup_parse_context_unref(ptr),
+        type_ => || ffi::g_markup_parse_context_get_type(),
+    }
+}
+
+impl MarkupParseContext {
+    //#[doc(alias = "g_markup_parse_context_new")]
+    //pub fn new(parser: /*Ignored*/&MarkupParser, flags: /*Ignored*/MarkupParseFlags, user_data: /*Unimplemented*/Option<Basic: Pointer>) -> MarkupParseContext {
```

Comment:
Is it useful without the constructor?

Reply:
Yes, you can create one with `pango_markup_parser_new` which is why I included it here. I can try to bind `g_markup_parse_context_new` in another PR